### DISCUSSION TITLE
Everyday Robot🦾

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#127] [General]
**新聞內容:**
Alphabet X於2019年11月公布通用型學習機器人專案Everyday Robot Project，旨在開發日常任務用機器人，未來還計畫能幫助獨居長者自理家務。使用攝影機及一組機器學習演算法來認知環境，並從中學習。
運用強化學習、協同學習的機器學習演算法及從示範中學習，Everyday Robot執行特定任務能力更成熟。以抓握小型物件為例，成功率由兩年前的75%進步到今天90%。在演算法改進下，當初機器人只能倒垃圾、拿杯子，現在也能拿抹布擦桌子，拿杯子的機器手也能開門。機器人甚至從開門的動作中學習，用來展開餐廳的椅子。
目前這些機器人已經在Alphabet大樓內提供服務，包括在擦桌子、打開會議室檢查是否少了椅子、並將椅子排列整齊。和機器人進展同步，本月Google公布多模態AI架構Pathways，具備將一種任務的模型泛化的能力，以用於學習其他任務，可加速學習並因應不斷出現的新需求。
**新聞心得:**
我覺得這種服務性質的機器人發展的功能越來越進步，運用在公司、餐廳上，不僅能增添科技感，更能減少處以雜事的人力成本，另外，未來發展的協助老人處理家務，因為許多年輕人無法每日照顧家中長輩、擔心他們的生活起居是否能自理，這時候機器人就能成為人類的小助手了！
**新聞來源:**
https://www.ithome.com.tw/news/147954
![f3uy4f7rjh2ds8h5xp38e696jhk1nc](https://user-images.githubusercontent.com/90883526/143028192-217229dd-95e2-42ae-b497-bf66109628a9.jpeg)

**Comment by pocper1** [comment]
我覺得現階段離家庭照顧機器人還有一段距離，因為家庭照顧所做的事情更為瑣碎。不過期望未來有家庭機器人的出現，這樣就不用做家事了XD

**Comment by Timmy110403540** [comment]
之前有印象去過機器人餐廳，許多工作都由機器人包辦，感覺很像在未來城市裡吃飯，很有科技感~

**Comment by briankuo123** [comment]
之前有看過會幫客人帶路的機器人，但使用後感覺還是有點可怕因為不知道他會突然做甚麼事情。可能以後會需要多花一點時間習慣。

**Reply by pocper1** [reply]
機器人帶路目前還是沒有人類的親切感～如果機器人有親切感或許會更好適應

**Comment by Jay-722** [comment]
可以幫忙倒垃圾感覺很不錯喔

**Comment by wh20502** [comment]
我覺得以後長照會是一大商機，如果可以設計機器人來陪伴老人或是照顧他，那麼真的會熱賣，高齡化社會逐漸來臨，年輕人打拼真的分身乏術。

**Comment by Shiba010** [comment]
如果有機器人能照顧老人，那將會是現代高齡社會下的福音！

**Comment by yasklop** [comment]
這種做雜事的機器人很適合我這種懶人，好想要有一台喔

**Comment by Yunhans** [comment]
懶人利器
